seed/static/seed/js/controllers/menu_controller.js

```diff
@@ -53,19 +53,16 @@ angular.module('BE.seed.controller.menu', [])
       };
       $scope.$on('app_error', function (event, data) {
         // Keep the first error
-        if (!$rootScope.route_load_error) {
-          $rootScope.route_load_error = true;
-          $scope.menu.error_message = data.message;
-          $rootScope.load_error_message = data.data.message;
-        }
+        $scope.menu.error_message = data.message;
+        $rootScope.load_error_message = data.data.message.join(' ');
+        if (!$rootScope.route_load_error) $rootScope.route_load_error = true;
```

Comment:
Maybe we can have a quick chat about this one tomorrow. I'm still not sure what exactly this does.

Reply:
Definitely. The point of doing this is to prevent confusion for users if there are multiple errors generated. As of now, we have 2 potential errors: valid rule without a label and must include/must not include an empty string.
<img width="868" alt="Screen Shot 2020-05-08 at 6 52 52 AM" src="https://user-images.githubusercontent.com/49968203/81407416-9982d980-90f8-11ea-94d3-ea3c560ee1d9.png">

Before, it shows only one error message if multiple errors found on page because on view/models/data_quality.py it returns which error message gets trigger first.

Reply:
@adrian-lara I think i see the purpose of 'Keep the first error' for this comment. Originally i was confused by not seeing all error messages when multiple errors were found on the page and that was the motive of modifying this piece of code. I will change it back to keeping the first error.

Reply:
Per our discussion, this will likely be updated in an upcoming PR.